content/blog/2020-04-usethis-1-6-0/index.Rmarkdown

```diff
@@ -0,0 +1,96 @@
+---
+title: usethis 1.6.0
+author: Hadley Wickham
+date: '2020-04-06'
+slug: usethis-1-6-0
+categories:
+  - package
+tags:
+  - devtools
+  - usethis
+photo:
+  author: Rock'n Roll Monkey
+  url: https://unsplash.com/photos/R4WCbazrD1g
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're happy to announce that [usethis](https://usethis.r-lib.org) 1.6.0 is now available on CRAN. usethis is a package that facilitates interactive workflows for R project creation and development. It's mostly focussed on easing day-to-day package development, but many of its functions are also useful for non-package projects. Install usethis with:
+
+```{r eval = FALSE}
+install.packages("usethis")
+```
+
+This blog post discusses three sets of improvements to usethis:
+
+* Our helpers for using GitHub actions are now polished, and if you're an 
```

Comment:
I feel like we should use the official capitalization, throughout: Github Actions.